src/views/design/ScriptsOverview/index.tsx

```diff
@@ -312,15 +317,16 @@ const ScriptsOverview = withStyles(styles)(
                     <Box paddingBottom={5} marginX={2.8}>
                         { !hasError && (
                             <GenericList
-                                listActions={[
-                                    {
+                                listActions={[].concat(
+                                    checkAuthority(SECURITY_PRIVILEGES.S_EXECUTION_REQUEST_WRITE, 'PUBLIC') ? {
                                         icon: <PlayArrowRounded />,
                                         label: translator('scripts.overview.list.actions.execute'),
                                         onClick: this.setScriptToExecute,
-                                    }, {
+                                    } : [],
+                                    checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC') ? {
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.